packages/fxa-profile-server/backstage.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: fxa-profile-server
+  description: Profile Server back-end for Mozilla Accounts that handles user profile data.
+  annotations:
+    sentry.io/project-slug: mozilla/fxa-profile
+    circleci.com/project-slug: github/mozilla/fxa
+  tags:
+    - javascript
+    - node
+    - hapi
+spec:
```

Comment:
Needs `providesApi`?

Reply:
The `providesApi` block requires a reference to an OpenAPI or GraphQL schema, which we don't appear to have.